content/articles/external-dns-providers.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: External DNS Providers
+excerpt: Link an external DNS provider to your DNSimple account to manage zones at other authoritative DNS providers, all within DNSimple.
+categories:
+- DNS
```

Comment:
I wonder if this should also be in `Integrations`.

Reply:
I think that's a great idea! Added it: https://github.com/dnsimple/dnsimple-support/pull/1032/commits/063a34d9040a1d9f1bbc0274f3e416404e633b69